src/model/movies.model.ts

```diff
@@ -1,8 +1,6 @@
-import { Thumbnails } from './api-response.model';
-
 export interface Movie {
     id?: number;
-    movieId: string;
+    movieId: string | number;
```

Comment:
skad ta zmiana? Movie to jest Twój typ, który uzywasz w appce wiec wydaje mi sie, ze powinienes miec jednolity ten typ